src/NuGet.Core/NuGet.LibraryModel/LibraryDependency.cs

```diff
@@ -39,18 +40,25 @@ public class LibraryDependency : IEquatable<LibraryDependency>
 
         public bool GeneratePathProperty { get; set; }
 
-        public string Aliases { get; set; }
+        public string? Aliases { get; set; }
 
         /// <summary>
         /// Gets or sets a value indicating a version override for any centrally defined version.
         /// </summary>
-        public VersionRange VersionOverride { get; set; }
+        public VersionRange? VersionOverride { get; set; }
 
         public LibraryDependency()
         {
             NoWarn = new List<NuGetLogCode>();
         }
 
+        [SetsRequiredMembers]
+        public LibraryDependency(LibraryRange libraryRange) : this()
```

Comment:
Should we add XML doc comments indicating that specifying a null `libraryRange` will throw an `ArgumentNullException`?  Or is that overkill here?

Reply:
I'm not seeing the text within any exception docs, including in other XMLDoc that already exists, in VS 2022 Int Preview, public preview, or GA versions 😕 
![image](https://github.com/NuGet/NuGet.Client/assets/5030577/4350cbcf-934d-48cc-9e26-39a8047ea09e)

Reply:
Interesting, I expected the tooltip of the Exception to show the text but yeah I can't figure out how the text is displayed in the IDE.  It must only be for docs? 🤷